editor/editor_node.cpp

```diff
@@ -7361,7 +7361,11 @@ EditorNode::EditorNode() {
 	vcs_actions_menu = VersionControlEditorPlugin::get_singleton()->get_version_control_actions_panel();
 	vcs_actions_menu->set_name("Version Control");
 	vcs_actions_menu->connect("index_pressed", callable_mp(this, &EditorNode::_version_control_menu_option));
-	vcs_actions_menu->add_item(TTR("Create Version Control Metadata..."), RUN_VCS_METADATA);
+	Ref<FileAccess> fileAccess = FileAccess::create(FileAccess::ACCESS_RESOURCES);
```

Comment:
```suggestion
	Ref<FileAccess> file_access = FileAccess::create(FileAccess::ACCESS_RESOURCES);
```
You should use `snake_case` for the code.